db/db_iter.cc

```diff
@@ -372,6 +372,7 @@ void DBIter::Next() {
   assert(valid_);
   assert(status_.ok());
 
+  PERF_CPU_TIMER_GUARD(iter_next_cpu_nanos, env_);
```

Comment:
does it make sense to use `iter_next_cpu_nanos` to track time spent on `Prev()` calls? or do you plan to introduce one more counter for that?

Reply:
@miasantreble no I didn't have any plan for `Prev()`. But I can add it to the commit using either way.